### ISSUE TITLE
What happens now that extensions need to be downloaded from the Chrome Webstore?

### ISSUE THREAD

**Issue by Jorys-Paulin** [moonlight-stream/moonlight-chrome#478]
Some days ago Google announced that Chrome extensions needed to be downloaded from the Chrome Webstore only, which mean we can't download Moonlight from GitHub directly.

This raised a few questions:
- Will users have to build the extensions themselves?
- Could we pack the crx in a zip to bypass the system?
- @cgutman, do you plan to publish on the webstore?

The extension's future is already threatened by NaCL and extensions deprecations, so I'm concerned by this announcement.

**Comment by stuarta0** [comment]
Confirmed. I can't download and install the CRX in a new Chrome install (67.0.3396.87).

**Comment by Jorys-Paulin** [comment]
@stuarta0 That might be a problem for most users. I'm running the git/unpacked version, so I'm not concerned yet, but others might have to also run the unpacked version...

**Comment by stuarta0** [comment]
Yep I'd update the releases description to include this method. Downloading the release source zip, extracting to folder, then using "Load Unpacked" on the Chrome extensions page with Developer Mode on works.

**Comment by Resolvy** [comment]
ye that works fine!!!

**Comment by apoklyps3** [comment]
I wouldn't worry about it.
Chrome extension seems it's getting abandonded anyway. It get's about 3-4 releases per year in comparition with the other ones with lots of commits never making it since no new version getting released.
we couldn't ask for more though and would like to thank @cgutman for all the support so far. never thought chrome would be the end of us, before nVidia :))

**Comment by Jorys-Paulin** [comment]
@apoklyps3 Yes, but recently I just contributed my biggest update yet. It was difficult since the code is kinda messy (after all that chrome extension was initially made in 24hrs), plus Google seems to kill everything it relies on (NaCL, Extensions, ...). So I started prototyping a new version of Moonlight, bundled with its own server, this time using WebRTC, widely supported across browsers. Plus it runs on Electron, meaning native support.

**Comment by SnekSnak** [comment]
@Jorys-Paulin See here > https://github.com/moonlight-stream/moonlight-chrome/issues/479#issuecomment-398114071

**Comment by apoklyps3** [comment]
@Jorys-Paulin Cool. Any chance you can share it ?

**Comment by Jorys-Paulin** [comment]
@apoklyps3 It's still in early development, I only did the screen capture for now, there's still the streaming and the client to do. Maybe I'll be asking for help...

**Comment by xXS4ALLx** [comment]
After pulling my hair out trying to install this, I stumbled on this issue post and managed to get it installed.

I noticed there are already several instructions on how to install, here is my take on it:

- Download the moonlight-chrome.crx file as per normal.
- I used 7zip to extract the contents to a folder.
- Enable Developer Mode on the Chrome Extensions page.
- Click on Load Unpacked and point to the folder I extracted the moonlight-chrome.crx file.
- Happy Days.
